week3/homework/src/actions.js

```diff
@@ -0,0 +1,88 @@
+'use strict';
+
+const { readFile, writeFile } = require('fs');
+const { promisify } = require('util');
+const uuid = require('uuid/v4');
+const read = promisify(readFile);
+const write = promisify(writeFile);
+const FILE_PATH = 'todo.json';
+const list = {};
```

Comment:
Why is this variable defined here?

Reply:
2 reasons :smile:   1. I followed the class example and 2. last week I passed the todo object directly to the list, inside the function, and during the review, Alexandre asked if I can bring it outside the function. So I thought that's what I needed to do here as well  :sweat_smile: especially when in the class example it was outside too. If that's wrong I guess I misunderstood Alexander's comment. Shall I bring it inside the function?

Reply:
What results do you get when you try to run your own code and add multiple to-dos? Try it and see if you get expected results.